### ISSUE TITLE
numpy 1.21.6 not compatible with Python 3.6

### ISSUE THREAD

**Issue by ahsan-ca** [ROCmSoftwarePlatform/AMDMIGraphX#2366]
The setup for python virtual environment fails for Python 3.6 because numpy 1.21.6 is not compatible with Python 3.6.
```
296/325 Test #315: py_3.6_common_setup_env ...................................................***Failed    1.61 sec
Collecting numpy==1.21.6 (from -r
/long_pathname_so_that_rpms_can_package_the_debug_info/src/extlibs/AMDMIGraphX/test/py/requirements.txt (line 25))
Could not find a version that satisfies the requirement numpy==1.21.6
(from -r /long_pathname_so_that_rpms_can_package_the_debug_info/src/extlibs/AMDMIGraphX/test/py/requirements.txt (line 25)) 
(from versions: 1.3.0, 1.4.1, 1.5.0, 1.5.1, 1.6.0, 1.6.1, 1.6.2, 1.7.0, 1.7.1, 1.7.2, 1.8.0, 1.8.1, 1.8.2, 1.9.0,
1.9.1, 1.9.2, 1.9.3, 1.10.0.post2, 1.10.1, 1.10.2, 1.10.4, 1.11.0, 1.11.1, 1.11.2, 1.11.3, 1.12.0, 1.12.1,
1.13.0, 1.13.1, 1.13.3, 1.14.0, 1.14.1, 1.14.2, 1.14.3, 1.14.4, 1.14.5, 1.14.6, 1.15.0, 1.15.1, 
1.15.2, 1.15.3, 1.15.4, 1.16.0, 1.16.1, 1.16.2, 1.16.3, 1.16.4, 1.16.5, 1.16.6, 1.17.0, 1.17.1,
1.17.2, 1.17.3, 1.17.4, 1.17.5, 1.18.0, 1.18.1, 1.18.2, 
1.18.3, 1.18.4, 1.18.5, 1.19.0, 1.19.1, 1.19.2, 1.19.3, 1.19.4, 1.19.5)
No matching distribution found for numpy==1.21.6 (from -r 
/long_pathname_so_that_rpms_can_package_the_debug_info/src/extlibs/AMDMIGraphX/test/py/requirements.txt (line 25))
```
Logs: https://compute-artifactory.amd.com/artifactory/rocm-qa-test-logs/test-logs/rocm-iv-3_810_2023-10-25_15:13/migraphx_810_48.log

We need to figure out if we want to use an older version of numpy for Python 3.6 or disable the tests on Python 3.6 like we did for Python onnx tests.

**Comment by ahsan-ca** [comment]
Discussed this in the meeting and got feedback to use an older version of numpy for testing across all Python versions.

**Comment by ahsan-ca** [comment]
PR is up for review.

**Comment by ahsan-ca** [comment]
PR approved and is ready to be merged in.

**Comment by ahsan-ca** [comment]
PR merged in, numpvy version has been set to 1.19.5.
